templates/master/00-master/openstack/files/openstack-keepalived-keepalived.yaml

```diff
@@ -9,7 +9,7 @@ contents:
         weight 50
     }
     vrrp_script chk_dns {
-        script "/usr/bin/host -t SRV _etcd-server-ssl._tcp.{{ .EtcdDiscoveryDomain }} localhost"
+        script "/usr/bin/host api-int.{{ .EtcdDiscoveryDomain }} localhost"
```

Comment:
should we use the https://github.com/openshift/api/blob/95abe2d2f4223d5931e418bf8e4d3773d16b42c0/config/v1/types_infrastructure.go#L74-L78

So that we are not assuming the `api-int` subdomain exists on `EtcdDiscoveryDomain`. Because that is technically not true or part of the API.

Reply:
This check no longer exists, cf https://github.com/openshift/machine-config-operator/pull/1666/